litestar/cli/_utils.py

```diff
@@ -123,7 +139,7 @@ class LoadedApp:
     is_factory: bool
 
 
-class LitestarGroup(Group):
+class LitestarGroup(Group):  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
You should be able to get around those typing errors by importing from click in an `if TYPE_CHECKING` block.

Reply:
Good callout. Can you take a look at the changes I made here?

Reply:
I left a comment there!